src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.js

```diff
@@ -48,51 +48,56 @@ function generator({ artifactTarball, versionTag, license, usePublicArtifact  })
   # REF: https://docs.openshift.org/latest/creating_images/guidelines.html
   RUN chmod -R g=u /usr/share/kibana
   RUN find /usr/share/kibana -type d -exec chmod g+s {} \\;
-  
+
   ################################################################################
   # Build stage 1
   # Copy prepared files from the previous stage and complete the image.
   ################################################################################
   FROM centos:7
   EXPOSE 5601
-  
+
   # Add Reporting dependencies.
-  RUN yum update -y && yum install -y fontconfig freetype && yum clean all
-  
+  RUN yum update -y && yum install -y fontconfig freetype wget && yum clean all
+
+  # Add an init process
+  RUN wget -O /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && chmod +x /usr/local/bin/dumb-init
```

Comment:
Core change is here. Didn't really find a good centos/yum repo to add for this, and the version is pinned, so hopefully OK for the time being.

Reply:
works for me, can we check the shasum too?

Reply:
Would probably be a good idea - releases are just tags which can be re-tagged.